benchexec/tools/coveriteam-verifier-validator.py

```diff
@@ -0,0 +1,32 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.coveriteam as coveriteam
+
+
+class Tool(coveriteam.Tool):
+    """
+    Tool info for a verifier or a validator based on
+    CoVeriTeam: a Configurable Software-Verification Platform.
+    URL: https://gitlab.com/sosy-lab/software/coveriteam.
+    """
+
+    def name(self):
+        return "Verifier (or Validator) Based on CoVeriTeam"
```

Comment:
This seems more like a description instead of a name. And it is very long and will probably disturb users when printed in the column headers of tables. Why not just `CoVeriTeam`?